tycho/context.py

```diff
@@ -41,31 +43,54 @@ class TychoContext:
     """
     
     """ https://github.com/heliumdatacommons/CommonsShare_AppStore/blob/master/CS_AppsStore/cloudtop_imagej/deployment.py """
-    def __init__(self, registry_config="app-registry.yaml", app_defaults_config="app-defaults.yaml",product="common", stub=False):
+    def __init__(self, registry_config="app-registry.yaml", app_defaults_config="app-defaults.yaml", product="common", tycho_config_url="", stub=False):
```

Comment:
you added the `tycho_config_url` parameter in the middle of the params instead of at the end. did you mean to add it at the end? have we checked to make sure no other places call this function with the old params order of `product` followed by `stub`, which would break if this new variable was added?

Reply:
my guess is that it is more likely that `tycho_config_url` is specified (not None).  Would need to track all these instances down though, should be easy.

Reply:
Probably would have been better to add it at the end, but it's only used in one place in Tycho (in ContextFactory, which is called by Appstore).  All the parameters are defined.

Reply:
Sounds good to me